Model and content for Coproporphyrin-I (#1)
Open-Systems-Pharmacology/Coproporphyrin-I: Evaluation/Input/Content/Section2.3_Model_Parameters_and_Assumptions.md

### 2.3.1 Absorption

> The parameter value for  `Specific intestinal permeability`  was set to 0 to not include intestinal reabsorption. The solubility was obtained from DrugBank (see [Section 2.2.1](#221-in-vitro-and-physicochemical-data))

Reviewer: What is the rationale for this? Why exclude intestinal reabsorption?.

Author: Added comment in the text (due to PIs going to very low value for this parameter and fixing it to the calculated value gave unreasonable fits)

> The synthesis rate was modelled as a zero-order infusion over one year ([Mochizuki 2022](#5-references)). 

Reviewer: Into liver cells, correct?

> The synthesis rate was modelled as a zero-order infusion over one year ([Mochizuki 2022](#5-references)). 

Reviewer: How was the rate selected? Why?

Author: Added in the text that it was (via MoBi)  based on the synthesis rate according to the paper (Mochizuki, 2022)

### 2.3.2 Distribution

Coproporphyrin-I is highly bound to plasma proteins (>99 %) (see [Section 2.2.1](#221-in-vitro-and-physicochemical-data)). A value of 0.66% was used in this PBPK model for `Fraction unbound (plasma, reference value)`. The major binding partner was set to albumin (see [Section 2.2.1](#221-in-vitro-and-physicochemical-data)).

> An important parameter influencing the resulting volume of distribution is lipophilicity. The reported logP value were 2.53 (see [Section 2.2.1](#221-in-vitro-and-physicochemical-data)) which was used in this model.

Reviewer: ```suggestion
An important parameter influencing the resulting volume of distribution is lipophilicity. The reported logP value of  2.53 (see [Section 2.2.1](#221-in-vitro-and-physicochemical-data)) is used in this model.
```

> After testing the available organ-plasma partition coefficient and cell permeability calculation methods built in PK-Sim, observed clinical data was best described by choosing the partition coefficient calculation by `Rodgers and Rowland` and cellular permeability calculation by `PK-Sim standard`.

Reviewer: Which observed data? The only data mentioned in this report is the plasma concentration, and as you are fitting the kcat, it is impossible to select the right partitioning method.

> After testing the available organ-plasma partition coefficient and cell permeability calculation methods built in PK-Sim, observed clinical data was best described by choosing the partition coefficient calculation by `Rodgers and Rowland` and cellular permeability calculation by `PK-Sim standard`.

Reviewer: Also, the partitionning method used in the model is "PK-Sim" and not R&R.

Author: Actually, in the simulations it is RR, but it was not selected as the default in the building block. Will update

### 2.3.3 Metabolism and Elimination

> Two transport protein was implemented into the model via Michaelis-Menten kinetics 

Reviewer: ```suggestion
Two transport proteins were implemented into the model via Michaelis-Menten kinetics - OATP1B1 and BCRP.
```

* OATP1B1

> The OATP1B1 expression profiles is based on high-sensitive real-time RT-PCR ([Nishimura 2003](#5-references)). The reference concentration for OATP1B1 was measured by liquid chromatography tandem mass spectroscopy ([Prasad 2013](#5-references)). Transporter activity was described as saturable process following Michaelis-Menten kinetics, were the `Km` was taken from literature and the `kcat` was optimized based on clinical data (see [Section 2.3.4](#234-automated-parameter-identification)).

Reviewer: ```suggestion
The OATP1B1 expression profile is based on high-sensitive real-time RT-PCR ([Nishimura 2003](#5-references)). The reference concentration for OATP1B1 was measured by liquid chromatography tandem mass spectroscopy ([Prasad 2013](#5-references)). Transporter activity was described as saturable process following Michaelis-Menten kinetics, were the `Km` was taken from literature and the `kcat` was optimized based on clinical data (see [Section 2.3.4](#234-automated-parameter-identification)).
```

> The OATP1B1 expression profiles is based on high-sensitive real-time RT-PCR ([Nishimura 2003](#5-references)). The reference concentration for OATP1B1 was measured by liquid chromatography tandem mass spectroscopy ([Prasad 2013](#5-references)). Transporter activity was described as saturable process following Michaelis-Menten kinetics, were the `Km` was taken from literature and the `kcat` was optimized based on clinical data (see [Section 2.3.4](#234-automated-parameter-identification)).

Reviewer: Does this mean that the reference concentration has been changed from the default value implemented in PK-Sim database? If so - make it clear.

Same for BCRP.

> * BCRP

Reviewer: Mention that the transporter is called "MRP2" in the model.

> The BCRP expression profiles isbased on high-sensitive real-time RT-PCR ([Nishimura 2003](#5-references)). The reference concentration for OATP1B1 was measured by liquid chromatography tandem mass spectroscopy ([Prasad 2014](#5-references)). Transporter activity was described as saturable process following Michaelis-Menten kinetics, were the `Km` was taken from literature and the `kcat` was optimized based on clinical data (see [Section 2.3.4](#234-automated-parameter-identification)).

Reviewer: Reference concentration of OATP in the BCRP section? Copy-paste or leftover?

> Additionally, renal clearance was set to 1 according to literature (see [Section 2.2.1](#221-in-vitro-and-physicochemical-data)).

Reviewer: What kind of renal clearance? Glomerular filtration with GFR 100% if so - why the assumption?

Author: Yes GFR fraction set to 1, it fit with the observed renal clearance rate as described in the literature - added comment in the text

### 2.3.4 Automated Parameter Identification

> The following parameters were optimized by fitting the model to the data:

Reviewer: How did you distinguish between the contribution of OATP and BCRP?

Author: As the compound is intracellularly synthesized and we had DDIs with probenecid only inhibiting OATP1B it was possible to distinguish the processes in this case
